frontend/views/components/FeedbackBanner.vue

```diff
@@ -0,0 +1,49 @@
+<template lang='pug'>
+  message.c-container(v-if='message' :severity='severity')
+    .c-inner
+      span {{ message }}
+      button.is-icon-small(
+        :aria-label='L("Dismiss banner")'
+        @click='dismiss'
+      )
+        i.icon-times
+</template>
+
+<script>
+import Message from '@components/message'
+
+export default {
+  name: 'FeedbackBanner',
+  components: {
+    Message
+  },
+  props: {
+    message: String,
+    severity: {
+      type: String,
+      default: 'danger',
+      validator: (v) => ['info', 'warning', 'danger'].includes(v)
+    }
+  },
+  methods: {
+    dismiss () {
+      this.$emit('update:message', '')
```

Comment:
Can you add a comment here linking to https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier ?